wordpress-mobile/WordPress-iOS
# Issues/7272 2fa screen

Refs #7272 

This PR apples the remaining visual style to the 2fa view controller and removes its inheritance from the old Signin2FAViewController.

Notes: 
As @hypest pointed in chat, the text in the comp was specific to someone using SMS for their auth code.  I've opted to match what is in the Android app currently rather than what's in the comp.  There might still be room for a slight tweak to the instructional text, so pinging for a copy review as well.
I've also left the status label in place tho I'm not sure we should keep it.  Seems like a hold over from the previous design and not a great fit here.  Thoughts? 

To test:
- Ensure the visual style was correctly implemented.
- Confirm logging in with a 2fa code works correctly.

Needs review: @nheagy 

Examples:

![simulator screen shot jun 28 2017 5 22 39 pm](https://user-images.githubusercontent.com/1435271/27664014-e862720a-5c2a-11e7-95ce-a5abd6be519b.png)
![simulator screen shot jun 28 2017 5 23 09 pm](https://user-images.githubusercontent.com/1435271/27664019-ec6a3464-5c2a-11e7-866b-e9c18b4f5b7f.png)
![simulator screen shot jun 28 2017 5 29 11 pm](https://user-images.githubusercontent.com/1435271/27664022-eee9c826-5c2a-11e7-8a7a-40cea02b5517.png)
